src/Sulu/Bundle/SecurityBundle/Controller/UserController.php

```diff
@@ -241,6 +263,16 @@ public function putAction(Request $request, $id)
             $user->setContact($this->getContact($request->get('contact')['id']));
             $user->setUsername($request->get('username'));
 
+            if ($request->get('email') !== null) {
```

Comment:
these code seems to be more or less duplicated with line 398 maybe you could extract it into an own function!